workflow/rules/utils.smk

```diff
@@ -33,4 +33,37 @@ rule tabix_known_variants:
         get_tabix_params
     cache: True
     wrapper:
-        "0.45.1/bio/tabix"
\ No newline at end of file
+        "0.45.1/bio/tabix"
+
+
+rule get_covered_regions:
+    input:
+        ref="resources/genome.fasta",
+        bam="results/recal/{sample}.sorted.bam",
+        bai="results/recal/{sample}.sorted.bam.bai"
+    output:
+        temp("results/regions/temp/{sample}.quantized.bed.gz")
+    params:
+        prefix=lambda wc, output: output[0].split(".quantized.bed.gz")[0]
+    shadow: "minimal"
+    log:
+        "logs/bam-regions/{sample}.log"
+    group: "covered-regions"
```

Comment:
```suggestion
```

Now the group is not a good idea anymore, because all the mosdepth runs of the same group would run in the same job.